src/util/__tests__/getServiceBaseURL.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env mocha */
+/* global expect, testContext */
+/* eslint-disable prefer-arrow-callback, no-unused-expressions */
+
+import getServiceBaseURL, {IDM, GAME} from '../getServiceBaseURL'
+
+describe(testContext(__filename), function () {
+  after(function () {
+    process.env.NODE_ENV = 'test'
+  })
+
+  it('throws an error if the service name is invalid', function () {
+    expect(() => getServiceBaseURL('xYz')).to.throw
+  })
+
+  it('returns the .dev service when in development', function () {
+    process.env.NODE_ENV = 'development'
+    expect(getServiceBaseURL(IDM)).to.equal('http://idm.learnersguild.dev')
+  })
+
+  it('returns the .org service when not in development', function () {
+    process.env.NODE_ENV = 'production'
```

Comment:
This is a big red flag for me. Setting the env to production in a test just seems scary. The code this is testing is simple enough that I'd probably rather just drop these tests altogether than have to change the NODE_ENV dynamically in a test to have to test them.

One other option would be to inject the env string as an optional second param and default it to `NODE_ENV`. That way you can test that path without changing the global environment.